src/screens/PodcastsScreen.tsx

```diff
@@ -147,6 +148,9 @@ export class PodcastsScreen extends React.Component<Props, State> {
         PV.Alerts.BUTTONS.OK
       )
     }
+
+    await gaInitialize()
```

Comment:
I think this should go in App.js instead of here. Its a global enough thing where it should probably be moved to the highest level component

Reply:
Ok done.